lib/factory_bot/decorator/invocation_tracker.rb

```diff
@@ -11,6 +11,10 @@ def method_missing(name, *args, &block)
         super
       end
 
+      def respond_to_missing?(*)
```

Comment:
If all we are doing is calling `super`, this seems unnecessary to me and probably slightly worse for performance because of an extra method call. How would you feel about ignoring this one?

Reply:
I'd be OK with that